cncf-youtube-channel-summarizer/extract_youtube_videos_info.py

```diff
@@ -0,0 +1,78 @@
+from googleapiclient.discovery import build
+from youtube_transcript_api import YouTubeTranscriptApi
+import json
+from tqdm import tqdm
+import os
+def get_channel_playlist(youtube, channel_id):
```

Comment:
Is it possible to improve this in future to pick only incremental playlists/videos added?  The idea is not summarize the same during every single run.